### PR TITLE
NOJIRA: Update cluster-etcd-operator e2e tests to use TechPreviewNoUp…

### PR DESCRIPTION
**Pull request by dusk125**
…grade

This should fix the issues with the installer on AWS

### CODE REVIEW

**Comment by hasbro17** [comment]
/lgtm

Although do we also need to update for 4.16 (and prior)?
https://github.com/openshift/release/blob/1b79448ab755d34655bb5e65ec4f2fca4cea2fd9/ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.16.yaml#L124-L141

I can't tell if our installs for e2e-operator are broken there too.

File: ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml [hunk 1]
```diff
@@ -138,7 +138,7 @@ tests:
     cluster_profile: aws
     env:
       FEATURE_GATES: '[EtcdBackendQuota=true,AutomatedEtcdBackup=true]'
```
Comment (tjungblu) [inline ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml:140]:
```suggestion
      - FEATURE_GATES: '[EtcdBackendQuota=true,AutomatedEtcdBackup=true]'
```

File: ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml [hunk 2]
```diff
@@ -121,7 +121,7 @@ tests:
...
```
Comment (tjungblu) [inline ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml:123]:
```suggestion
```

**Pull request merged**